### ISSUE TITLE
null pointer in ProtocolManager

### ISSUE THREAD

**Issue by splix** [ethereumproject/go-ethereum#375]
v4.0.0, failed just after the start

```
I1020 23:19:49.347735 eth/downloader/downloader.go:344] Block synchronisation started
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5150de]

goroutine 9149 [running]:
math/big.(*Int).Cmp(0xc42e1884a0, 0x0, 0x4b297603a73d1d4b)
        /home/travis/.gimme/versions/go1.8.linux.amd64/src/math/big/int.go:314 +0x2e
github.com/ethereumproject/go-ethereum/eth.(*ProtocolManager).synchronise(0xc4202e5380, 0xc4225d1d40)
        /home/travis/gopath/src/github.com/ethereumproject/go-ethereum/eth/sync.go:174 +0x11e
github.com/ethereumproject/go-ethereum/eth.(*ProtocolManager).syncer.func1()
        /home/travis/gopath/src/github.com/ethereumproject/go-ethereum/eth/sync.go:135 +0x4b
created by github.com/ethereumproject/go-ethereum/eth.(*ProtocolManager).syncer
        /home/travis/gopath/src/github.com/ethereumproject/go-ethereum/eth/sync.go:150 +0x23f
```

**Comment by splix** [comment]
Actually it was because `no space left on device`, but nil pointer doesn't look good anyway

**Comment by splix** [comment]
The problem that after disk resize it started to sync, but from genesis (lost 4M+ of dowloaded chain). rel #274

**Comment by whilei** [comment]
Fixed with #379
